readme.md

```diff
@@ -22,6 +24,10 @@ a roots extension for inlining your CSS properties into the style attribute in a
     extensions: [ rootsInlineCss(options) ]
   ```
 
+  This extension provides a way for you to inline your html with css. By default all of your `html` files will be inlined however you can specify through options which files you want inlined either with a single file or a [glob expression]( https://github.com/isaacs/minimatch).
```

Comment:
I noticed that arrays and hashes have a space between the bracket and element. Is this for readability?
`[ rootsInlineCss(options) ]` as an example.